public/.htaccess

```diff
@@ -0,0 +1,21 @@
+<IfModule mod_rewrite.c>
```

Comment:
For what we need `.htaccess` if there is no `Apache` server?

Reply:
removed